release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,932 @@
+programming languages and frameworks, such as Java, Ruby, and PHP.
+
+Built on Red Hat Enterprise Linux and Google Kubernetes, {product-title}
+provides a secure and scalable multi-tenant operating system for today’s
+enterprise-class applications, while providing integrated application runtimes
+and libraries. {product-title} brings the OpenShift PaaS platform to customer
+data centers, enabling organizations to implement a private PaaS that meets
+security, privacy, compliance, and governance requirements.
+
+[[ocp-39-about-this-release]]
+== About This Release
+
+Red Hat {product-title} version 3.9
+(link:https://access.redhat.com/errata/RHBA-2018:0489[RHBA-2018:0489) is now
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v3.9.0-rc.0[OpenShift
+Origin 3.9]. New features, changes, bug fixes, and known issues that pertain to
+{product-title} 3.9 are included in this topic.
+
+To better synchronize versions of {product-title} with Kubernetes, Red Hat did
+not publicly release {product-title} 3.8 and, instead, is releasing
+{product-title} 3.9 directly after version 3.7. See
+xref:ocp-39-installation[Installation] for information on how this impacts
+installation and upgrade processes.
+
+{product-title} 3.9 is supported on RHEL 7.3, 7.4, 7.5 and Atomic Host 7.4.5 and
+newer with the latest packages from Extras, including Docker 1.13. The
+*Docker-latest* package is now deprecated.
+
+For initial installations, see the
+xref:../install_config/install/planning.adoc#install-config-install-planning[Installing
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+To upgrade to this release from a previous version, see the
+xref:../install_config/upgrading/index.adoc#install-config-upgrading-index[Upgrading
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+[[ocp-39-new-features-and-enhancements]]
+== New Features and Enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[[ocp-39-container-orchestration]]
+=== Container Orchestration
+
+[[ocp-39-soft-image-pruning]]
+==== Soft Image Pruning
+
+Now, when pruning images, you do not have to remove the actual image, just
+update etcd storage.
+
+It is safer to run `--keep-tag-revisions` and `--keep-younger-than`. After this
+is run, administrators can choose to run hard prune (which is safe to run as
+long as the registry is put in read-only mode).
+
+[[ocp-39-cloudforms]]
+==== CloudForms 4.6 Container Management
+
+* {product-title} template provisioning
+* Offline OpenScapScans
+* Alert management: You can choose Prometheus (currently in Technology Preview) and use it in CloudForms.
+* Reporting enhancements
+* Provider updates
+* Chargeback enhancements
+* UX enhancememts
+
+[[ocp-39-crio]]
+==== CRI-O v1.9 (Technology Preview)
```

Comment:
So this should change in 3.9.z to full support, not sure if I like this language given it will change so soon